Makefile

```diff
@@ -294,7 +294,7 @@ image-mirror-setup:
 	$(MAKE) apply WHAT=cluster/ci/jobs/image-mirror.yaml
 .PHONY: image-mirror-setup
 
-knative-image-mirror-setup:
+knative-image-miror-setup:
```

Comment:
typo?

Reply:
more like fat-fingering a delete key somewhere inadvertently. let me fix it.